juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule

```diff
@@ -5,96 +5,97 @@ healthbot {
+                iAgent {
+                    file vport.yml;
+                    table VirtualVportTable;
+                    frequency 60s;
+                }
             }
-        }
-        field vport-count {
-            formula {
-                user-defined-function {
-                    function-name difference;
-                    argument subtract_from "$vport_count";
-                    argument subtract 1;
+            field vport-count {
+                formula {
+                    user-defined-function {
+                        function-name difference;
+                        argument subtract_from "$vport_count";
+                        argument subtract 1;
+                    }
                 }
+                type integer;
             }
-            type integer;
-        }
-    field vport-percentage-used {
-            formula {
-                user-defined-function {
-                    function-name used-percentage;
-                    argument used "$vport-count";
-                    argument total "{{vport_max_count}}";
+            field vport-percentage-used {
+                formula {
+                    user-defined-function {
+                        function-name used-percentage;
+                        argument used "$vport-count";
+                        argument total "{{vport_max_count}}";
+                    }
                 }
+                type integer;
             }
-            type integer;
-        }
-        trigger vport-usage {
+            trigger vport-usage {
                 synopsis "vport kpi";
                 description "Sets health based on vport used percentage.";
-            term below-threshold {
-                when {
-                    less-than "$vport-percentage-used" 60;
-                }
-                then {
-                    status {
-                        color green;
-                        message " vport count is $vport-count which is  $vport-percentage-used  %of max";
+                frequency 90s;
+                term below-threshold {
+                    when {
+                        less-than "$vport-percentage-used" 60;
+                    }
+                    then {
+                        status {
+                            color green;
+                            message " vport count is $vport-count which is  $vport-percentage-used  %of max";
+                        }
                     }
                 }
-            }
-            term above-threshold {
-                when {
-                    greater-than "$vport-percentage-used" 80;
+                term above-threshold {
+                    when {
+                        greater-than "$vport-percentage-used" 80;
+                    }
+                    then {
+                        status {
+                            color red;
+                            message " vport count is $vport-count which is  $vport-percentage-used % of max";
```

Comment:
Remove space in begin of message
s/" vport count is $vport-count which is $vport-percentage-used %of max";/"Vport count($vport-count) and used percentage($vport-percentage-used %) is abnormal";/g

Reply:
updated this file